pkg/agent/datamodel/sig_config.go

```diff
@@ -434,7 +435,7 @@ var (
 		ResourceGroup: AKSWindowsResourceGroup,
 		Gallery:       AKSWindowsGalleryName,
 		Definition:    "windows-2022-containerd-gen2",
-		Version:       "20348.1006.220930", // TODO (wanqingfu, ShiqianTao) Workaround. Will improve it.
+		Version:       Windows2022Gen2SIGImageVersion, // TODO (wanqingfu, ShiqianTao) Improving by adding a new Gen 2 version.
```

Comment:
Shall I remove the comment?

Reply:
Please reference this commit to update them.
https://github.com/Azure/AgentBaker/commit/7a817661b1ff8773f65826a99fe117003b6d0d1e

Reply:
referenced.

Reply:
https://github.com/Azure/AgentBaker/pull/2339/commits/e24f629f2a897cd1b4d530ad4dae1209290360fe